copter/source/docs/systemid-mode-ctrl-optimization.rst

```diff
@@ -0,0 +1,257 @@
+
+Optimization Goals
+------------------
+
+The following table shows the goals/requirements of the controller optimization.
+The dynamics of the roll and pitch axes are very similar.
+Therefore, identical goals are applied to both axes.
+The yaw axis behaves quite differently.
+It is assumed that disturbances due to translational motion and wind mainly influence the pitch and roll axes, hence the disturbance rejection is not optimized for the yaw axis.
+
++------------------------------------+-----------------+---------------+----------------------+
+| Requirement Goal                   | Pitch & Roll    | Yaw           | Optimization Target  |
++------------------------------------+-----------------+---------------+----------------------+
+| Phase Margin                       | 50°             | 50°           | Larger is better     |
++------------------------------------+-----------------+---------------+----------------------+
+| Gain Margin                        | 7 dB            | 7 dB          | Larger is better     |
++------------------------------------+-----------------+---------------+----------------------+
+| Step Response: Response Time       | 0.02 s          | 0.2s          | Smaller is better    |
++------------------------------------+-----------------+---------------+----------------------+
+| Step Response: Overshoot           | 5 %             | 5 %           | Smaller is better    |
++------------------------------------+-----------------+---------------+----------------------+
+| Step Response: Stationary Error    | 5 %             | 0.1 %         | Smaller is better    |
++------------------------------------+-----------------+---------------+----------------------+
+| Control Variable Max. Overshoot    | 80 %            | 10 %          | Smaller is better    |
++------------------------------------+-----------------+---------------+----------------------+
+| Disturbance Rejection              | see graph below |not applied    | Smaller is better    |
++------------------------------------+-----------------+---------------+----------------------+
+
+The frequency-based disturbance rejection curve in decibel is shown below.
+As a reference, the blue line represents the rejection gain that is achieved with the default parameters of ArduCopter.
+The optimized system is required to stay bellow the yellow area to have a better disturbance rejection.
+
+.. image:: ../images/disturbanceRejectionRequirement.png
+:name: fig-dist-reject-req
+
+Optimization Results in the Simulation
+--------------------------------------
+
+The next four plots show the bode plot of the closed-loop system the simulation results for the tracking behavior of a 10 degree step, the normalized controller output corresponding to the angle step as well as the disturbance behavior for the roll axis.
+The normalized controller output evaluates whether the optimized behavior can actually be realized by the real system.
```

Comment:
I would suggest you do a better job at describing how the plot shows that the optimized behavior can realized by a real system.  It's not obvious to me.